### DISCUSSION TITLE
Cannot find the Curve Extract node for DNA/RNA

### DISCUSSION THREAD

**Discussion by famrein** [BradyAJohnston/MolecularNodes#273] [Q&A] [answered]
Hello! I was following the DNA in molecular nodes tutorial, and at around 16:00 theres a mention of a curve extract node (and some other nodes that I also can't see, like curve join, primitives, etc). I'm using the latest molecular nodes (2.7.4). Is this related to the "Current Limitations" section of the 2.0.1 release? Is this one of the nodes not yet ported over? Could I revert to the earlier molecular nodes just to get this node or is it just not worth it? Thanks!

**Comment by BradyAJohnston** [comment, marked as answer]
I don't think the old nodes would work with the new system but I haven't tried. I haven't yet ported over all of the nodes, but I am also currently in the process of overhauling a bunch of the internals for the DNA / RNA system. After that I will be porting thre remaining nodes and hopefully adding some more. Apologies for the lacking nodes so far, I know a bunch of people have been confused by that DNA video with the change in nodes that are available

**Reply by famrein** [reply]
That's what I thought, its no problem at all. I'm just thankful for all your work! Looking forward to the nucleic acid nodes 🤗
